src/app/triggerFromForm.ts

```diff
@@ -9,14 +11,14 @@ interface TriggerFromFrom {
 }
 
 export class TriggerFromFormImpl implements TriggerFromFrom {
-    values: AppFormValues;
+    values: AppCallValues;
     context: AppContext;
-    trigger: any
+    trigger: ZDTrigger
 
-    constructor(call: AppCall) {
-        this.values = call.values as AppFormValues;
+    constructor(call: AppCallRequest) {
+        this.values = call.values as AppCallValues;
```

Comment:
Why is this needed?

Reply:
Unfortunately. See comment https://github.com/mattermost/mattermost-app-zendesk/pull/34/files#r612100603

I suppose I could create `AppCallRequestWithValues` type that has them as a required field. What do you think?